app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,35 @@
+
+require_relative '../../lib/TunesTakeoutWrapper'
+
+
+class SuggestionsController < ApplicationController
+  attr_reader :show
+
+  def index
+    @data = TunesTakeoutWrapper.top_twenty
+  end
```

Comment:
Good clear name for this method. I know immediately what it's going to be showing.